src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/atclauseorder/InputAtclauseOrderIncorrectCustom.java

```diff
@@ -0,0 +1,301 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.atclauseorder;
+/** Javadoc for import */
+import java.io.Serializable;
+
+/* Config:
+ * target = "CLASS_DEF"
+ * tagOrder = "@since, @version, @param, @return, @throws, @exception, @deprecated, @see,
+ *          @serial, @serialField, @serialData,@author"
+ */
```

Comment:
It would be better if the file starts with configuration.
Like in all the input test cases of indentation, we are enforcing configs to be placed at the top of the file, so we can follow the same here to be more consistent.

Reply:
From https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-create-Input-files-for-tests: 
```java

//non-compiled with javac: Compilable with Java14
package com.puppycrawl.tools.checkstyle.checks.naming.recordcomponentname;

import java.io.Serializable;
import java.util.LinkedHashMap;

import org.w3c.dom.Node;

/* Config:
 * pattern = "^[a-z\\d]+$"
 */
public record InputRecordComponentNameLowercase(Integer x, String str, Double val123) { // ok
}
...
```
We should update the wiki example if this is the case.

Reply:
It is confirmed that all the `inputIndentation*` files requires their config to be written at the start of the file, and I like that ide too, since we already get a feel of what the config is and how the input file would look like.

Reply:
Issue opened at https://github.com/checkstyle/checkstyle/issues/9620

Reply:
@Abhishek-kumar09 we can revisit this format later: https://github.com/checkstyle/checkstyle/issues/9620

Please resolve this item for now.

Reply:
resolved